### PR TITLE
Update Install Instructions for MyNode

### PR DESCRIPTION
**Pull request by tehelsper**

### CODE REVIEW

File: docs/software/installation.md [hunk 1]
```diff
@@ -65,6 +66,15 @@ We hope that Jam will be available as a one-click app
 install with [RaspiBlitz
 v1.8.0](https://github.com/rootzoll/raspiblitz/issues/2891).
 
+### ...with MyNode
+
+Jam can be installed directly from the [MyNode](https://mynodebtc.com/) Marketplace:
+
+- Open the interface of your MyNode device <br/> (type `mynode.local` in your browser)
```
Comment (dergigi) [inline docs/software/installation.md]:
```suggestion
- Open the interface of your MyNode device <br/>(type `mynode.local` in your browser)
```

File: docs/software/installation.md [hunk 2]
```diff
@@ -65,6 +66,15 @@ We hope that Jam will be available as a one-click app
...
+- Find "Jam" in the Markplace (Note: The JoinMarket app must also be installed)
```
Comment (dergigi) [inline docs/software/installation.md]:
```suggestion
- Find "Jam" in the Markplace <br/>(Note: the JoinMarket app must also be installed)
```

**Review by dergigi — APPROVED** [review]
Awesome! Thank you for your contribution! I haven't tested this myself yet, but I assume you did when working on https://github.com/mynodebtc/mynode/pull/714 

Not sure if I'll get to it soon, but we can always update the docs again in case someone doesn't work as expected.

TL;DR: Looks good to me, utACK ✅
